app/views/newsletters/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="newsletter-text">
+  <p>
+    Want to be kept up to date with the latest news from Any Rat Rescue? 
+  </p>
+  <button class="subscribe"><%= link_to "Subscribe", "#" %></button>
```

Comment:
Since you have `bourbon` in this project, can you just use the `button` class on the `link_to` instead of wrapping it in a button element?